server/src/lib/server.rs

```diff
@@ -936,6 +959,45 @@ async fn instance_migrate_status(
         .map(HttpResponseOk)
 }
 
+#[derive(Deserialize, JsonSchema)]
+struct SnapshotRequestPathParams {
```

Comment:
Move to client::api since that's generally where we kept those before we removed the redundant IDs from most endpoints.

Reply:
done in c341b5d